src/mame/machine/k573kara.cpp

```diff
@@ -0,0 +1,172 @@
+// license:BSD-3-Clause
+// copyright-holders:windyfairy
+#include "emu.h"
+#include "k573kara.h"
+
+#define LOG_GENERAL    (1 << 0)
+// #define VERBOSE        (LOG_GENERAL)
+// #define LOG_OUTPUT_STREAM std::cout
+
+#include "logmacro.h"
```

Comment:
This should come after all the other headers,

Reply:
Looking at it now, I'm not using logging in this file anymore so I'll just remove it.